harvard-university-for-the-creative-arts.csl

```diff
@@ -191,13 +227,14 @@
           <text term="in" text-case="capitalize-first" suffix=": "/>
           <names variable="editor" delimiter="." suffix=" ">
             <name and="text" delimiter-precedes-last="never" et-al-min="3" et-al-use-first="1" initialize-with="." name-as-sort-order="all"/>
-            <label form="short" plural="never" prefix=" (" suffix=")"/>
+            <label form="short" plural="never" text-case="lowercase" prefix=" (" suffix=")"/>
             <et-al font-style="italic"/>
           </names>
           <group delimiter=", ">
             <group font-style="normal" font-variant="normal">
               <text variable="container-title" font-style="italic" suffix="."/>
               <text variable="collection-title" font-style="italic"/>
+              <text variable="volume" prefix=" Vol " suffix="."/>
```

Comment:
Thanks for the tip. I've added the way you suggested, I think - could you let me know if there is an issue? I've also added it in Book as we quite a few books of multiple volumes where it may also be used